shiny/experimental/ui/_page.py

```diff
@@ -20,10 +20,66 @@
 from ._css_unit import CssUnit, validate_css_padding, validate_css_unit
 from ._fill import as_fillable_container
 from ._navs import navset_bar
-from ._sidebar import Sidebar
+from ._sidebar import Sidebar, layout_sidebar
 from ._utils import consolidate_attrs
 
 
+def page_sidebar(
+    *args: TagChild | TagAttrs,
```

Comment:
@schloerke double-check the type on this (and pass `kwargs` to `layout_sidebar()`

```suggestion
    *args: TagChild | TagAttrs,
    **kwargs: TagAttrValue | CssUnit,
```